src/Bindings/Plugin.cpp

```diff
@@ -28,9 +28,33 @@ cPlugin::~cPlugin()
 
 
 
+void cPlugin::Unload(void)
+{
+	auto pm = cPluginManager::Get();
+	pm->RemovePluginCommands(this);
+	pm->RemovePluginConsoleCommands(this);
+	pm->RemoveHooks(this);
```

Comment:
You're right, they aren't being removed. This doesn't make a problem when reloading, since the tab names are overwritten, but it does make a problem when unloading a plugin via the `unload` console command and then going through webadmin.